www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/+TARGETS

```diff
@@ -2,3 +2,8 @@ c_icap:/etc/rc.conf.d/c_icap
 c-icap.conf:/usr/local/etc/c-icap/c-icap.conf
 newsyslog.conf:/etc/newsyslog.conf.d/c-icap
 virus_scan.conf:/usr/local/etc/c-icap/virus_scan.conf
+VIRUS_FOUND:/usr/local/etc/c-icap/templates/virus_scan/en/VIRUS_FOUND
```

Comment:
Are these files normally provided by c-icap ?

Reply:
Yep, they are usually in /usr/local/share/c_icap (see conf templating) but I don't want to touch them there.

Reply:
Can we only template what needs a template and move the template target to /tmp ? It would allow us to remove / revoke these files easily because otherwise they would be there forever